modules/oe_theme_content_entity_contact/oe_theme_content_entity_contact.install

```diff
@@ -20,18 +19,10 @@ function oe_theme_content_entity_contact_install() {
   }
 
   // Override entity view displays.
-  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_entity_contact') . '/config/overrides');
-  $displays = [
-    'core.entity_view_display.oe_contact.oe_general.default',
-    'core.entity_view_display.oe_contact.oe_press.default',
-  ];
-
-  foreach ($displays as $display) {
-    $values = $storage->read($display);
-    $config = EntityViewDisplay::load($values['id']);
-    foreach ($values as $key => $value) {
-      $config->set($key, $value);
-    }
-    $config->save();
-  }
+  $file_storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_entity_contact') . '/config/overrides');
```

Comment:
Proposed solution for https://citnet.tech.ec.europa.eu/CITnet/jira/browse/D8TEWPPE-53